introduction.md

```diff
@@ -0,0 +1 @@
+This repo is a workspace for the Mastering GitHub course in Code School
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.